Jenkinsfile

```diff
@@ -0,0 +1,99 @@
+#!groovy
+
+properties([buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '5')),
+            pipelineTriggers([]),
+            disableConcurrentBuilds()])
+
+def PRODUCT = 'bibliotekdk'
+// remove 'feature/' from BRANCH_NAME
+def BRANCH = BRANCH_NAME.replaceAll('feature/', '')
+// this is a static installation while we wait for the docker world
+def WWW_PATH = '/data/www/'
+// postgres database to use for bibliotek.dk installation
+def PG_NAME = "feature_${BRANCH}"
+// path to npm
+def NPM_PATH = "${WWW_PATH}${BRANCH}/profiles/bibdk/themes/bibdk_theme/.npm/"
+
+
+node('dscrum-is-builder-i01'){
```

Comment:
Jeg kender ikke kode syntaksen for groovy, så jeg ved ikke om der skal være et mellemrum her?